packages/core/src/system.ts

```diff
@@ -34,6 +37,17 @@ export function createSystem<T extends ActorSystemInfo>(): ActorSystem<T> {
 
       keyedActors.set(systemId, actorRef);
       reverseKeyedActors.set(actorRef, systemId);
+    },
+    get deadLetters() {
```

Comment:
I would prefer not to have getters with side-effects - and for that matter, i think it's best to avoid getters (they obscure the nature of a thing, when i debug and read code like `smth.foo` I don't anticipate that behind this banana is a gorilla holding the banana and the entire jungle)